pkgs/applications/video/obs-studio/plugins/obs-vkcapture.nix

```diff
@@ -13,13 +13,13 @@
 
 stdenv.mkDerivation rec {
   pname = "obs-vkcapture";
-  version = "1.1.3";
+  version = "1.1.4";
 
   src = fetchFromGitHub {
     owner = "nowrep";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-iIV9ke2yPEt2Lf4bwiEHFip4tLhMS4raWGyCWpao74w=";
+    sha256 = "sha256-jY78+sfAd62YnCssosLOLfxmdL6zkTEoeE58bQpswG4=";
```

Comment:
Hello, thanks for the review? Is there a reason to not use sha256 directly?

Reply:
> Hello, thanks for the review? Is there a reason to not use sha256 directly?

https://nixos.wiki/wiki/Nix_Hash#Usage
Since an SRI hash is used it would be better to use "hash" instead of "sha256"

Reply:
Hm, well the generated code is the same, but if you insist on it, I guess I don't mind changing it.

Reply:
done